packages/cli-e2e/__tests__/atomic.specs.ts

```diff
@@ -151,7 +178,7 @@ describe('ui:create:atomic', () => {
         'when using an existing pageId (using the list prompt of available pages)',
       buildAppOptions: {
         id: 'with-page-id-prompt',
-        promptAnswer: `\x1B[B ${EOL}`,
+        selectWithPrompt: true,
```

Comment:
Should there be a `pageIdOrName` here too?

Reply:
yep, changed and fixed